.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     docker:
       - image: node
     environment:
-      NPM_VERSION: "6.7.0"
+      NPM_VERSION: "5.7.0"
```

Comment:
Why did you change to an older npm?

Reply:
Because latest npm failed on node 6 ... since we're only going to use node six for another month, it'll be fine :)